src/components/TheHeader.vue

```diff
@@ -2,15 +2,27 @@
 import LogoImage from "@/assets/images/logo.svg?component";
 import HeaderActions from "@/components/header/HeaderActions.vue";
 import HeaderLinks from "@/components/header/HeaderLinks.vue";
+import HeaderBanner from "@/components/header/HeaderBanner.vue";
+import { computed } from "vue";
+import { useRoute } from "vue-router";
+const route = useRoute();
+
+const showActions = computed(() => {
+  return route.matched.some((el) => el.path !== "/login");
+});
+const showBanner = computed(() => {
+  return route.matched.some((el) => el.path === "/login");
+});
```

Comment:
Since they basically check for the same thing (if there's 'login'), you could just have one and  use `!` to negate it.

Reply:
I did this in on purpose as this rules will grow as header variant come in in next steps of app development.